ass2/index.html

```diff
@@ -0,0 +1,150 @@
+  </head>
+
+  <body>
+    <div class="navbar-container">
+    <nav class="navbar">
+      <a href="#!" class="brand-logo"><img src="img/logo.png"></a>
+      <div class="menu"><i class="icon-menu"></i></div>
+      <div class="search-container">
+        <form>
+        <label class="icon-search"></label>
+          <input type="text" name="search" placeholder="Lookup User">
+        </form>
+      </div>
+      <div class="clear"></div>
+  </nav>
+  </div>
+
+  <section class="sidebar">
+    <ul class="sidebar-options">
+      <li class="sidebar-name">
+        <div><span>J</span></div>
+        <h2>John Snow </h2>
+        <p>Acme corp</p>
+      </li>
+      <li class="sidebar-menu">Acme corp <i class="icon-share right"></i></li>
+      <li class="sidebar-menu"><i class="icon-briefcase"></i>Organization info</li>
+      <li class="sidebar-menu"><i class="icon-price-tags"></i>Tags</li>
+      <li class="sidebar-menu"><i class="icon-circle-down"></i>Aggregation Units</li>
+      <li class="sidebar-menu"><i class="icon-file-text2"></i>Reports</li>
+      <button class="sidebar-logout flat-button"><i class="icon-switch"></i>Logout</button>
+    </ul>
+  </section>
+
+  <section class="container">
+    <section class="landing-page">
+    <h2 class="landing-heading">List of members in <span>Acme Corp </span>
+    <button class="download"><i class="icon-download"></i></button></h2>
+    <button class="add-member button">Add member</button>
+    <button class="flat-button">View pending invitations</button>
+    <div class="dropdown">
+    <h6>Jump to</h6>
+      <span>
+      <select>
+        <option>J</option>
+        <option>c</option>
+      </select>
+      <span>
+    </div>
+    <div class="clear"></div>
+    <ul class="members-list">
+    <li>A</li>
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">A Stark</p>
+          <p class="user-email">astark@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">Agtha Alev</p>
+          <p class="user-email">agtha@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">Alec Applelbuam</p>
+          <p class="user-email">alec@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+    </ul>
+    <ul class="members-list">
+    <li>J</li>
+      <li> 
+        <div class="letter-style"><h3>J</h3></div>
+        <div class="user-info">
+          <p class="user-name">John Doe (owner)</p>
+          <p class="user-email">jdoe@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
```

Comment:
@priyanka0901 Common styling classes should come first then cutome class or color class
For Ex:
<button class="button make-owner">Make owner </button>